openapi/components/schemas/Storefront/StorefrontBillingPortal.yaml

```
@@ -4,10 +4,11 @@ required:
   - slug
 properties:
   id:
+    type: string
     description: ID of the billing portal.
     readOnly: true
-    allOf:
-      - $ref: ../ResourceId.yaml
+    maxLength: 50
+    example: bill_prt_0YV7K5TYV5D1P9ZNKDT39KZC3C
   token:
     description: Session's token used for authentication.
```

nitpick: [Possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)
```suggestion
    description: Token used for session authentication.
```